lib/portlayer/exec/container.go

```diff
@@ -555,6 +558,37 @@ func (c *Container) Remove(ctx context.Context, sess *session.Session) error {
 	return nil
 }
 
+// Update the VM display name on vSphere UI
+func (c *Container) UpdateDisplayName(ctx context.Context) error {
```

Comment:
This may not be needed if we can just do the rename as part of a reconfigure (which is done by `Commit`).